models/boards.js

```diff
@@ -170,6 +174,7 @@ Boards.mutations({
   },
 
   addMember(memberId) {
+    if (memberId === Meteor.userId()) return null;
```

Comment:
The security checks aren’t actually implemented in these `mutations` methods but using the allow/deny rules (because it is possible to overpass mutations checks by simply executing the minimongo query on the browser console). I know that this is a bit confusing and I’m considering an evolution of the whole mutation model (see also https://forums.meteor.com/t/new-package-mdg-method-meteor-methods-with-better-scoping-argument-checking-and-good-defaults/13833)